src/pages/TabsPage.tsx

```diff
@@ -0,0 +1,31 @@
+import { useParams } from 'react-router-dom';
+import { Tabs } from '../components/Tabs';
+
+const tabs = [
+  { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
+  { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
+  { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
+];
+
+export const TabsPage: React.FC = () => {
+  const { tabId } = useParams();
+
+  const selectedTab = tabs.find(({ id }) => id === tabId) || null;
+
+  return (
+    <>
+      <h1 className="title">Tabs page</h1>
+
+      <Tabs
+        tabs={tabs}
+        selectedTab={tabId}
```

Comment:
Since you pass only `id` here, you might want to choose a more representative name.
```suggestion
        selectedTabId={tabId}
```